src/main/java/bio/terra/grammar/google/BigQueryVisitor.java

```diff
@@ -39,16 +40,21 @@ public String visitColumn_expr(SQLParser.Column_exprContext ctx) {
     return String.format("`%s`.%s", alias, columnName);
   }
 
-  public static TableNameGenerator bqTableName(DatasetModel dataset) {
-    return (tableName) -> generateTableName(dataset, dataset.getName(), tableName);
+  public static TableNameGenerator bqSnapshotTableName(SnapshotModel snapshot) {
+    return (tableName) -> generateSnapshotTableName(snapshot, snapshot.getName(), tableName);
```

Comment:
parens are unnecessary for a single argument lambda
```suggestion
    return tableName -> generateSnapshotTableName(snapshot, snapshot.getName(), tableName);
```